runtime/src/serde_snapshot/tests.rs

```diff
@@ -262,7 +262,7 @@ mod test_bank_serialize {
 
     // These some what long test harness is required to freeze the ABI of
     // Bank's serialization due to versioned nature
-    #[frozen_abi(digest = "5rd8RyVSLH3hm12xJDVCJWgc1gyqb4Ukt2hJLJNfsB5v")]
+    #[frozen_abi(digest = "ULV2jDndxR3JB677ayyjaamtAcZ24q75tCkHS2bKVoy")]
```

Comment:
This hash changed too.  Again seems harmless but 🤔

Reply:
yeah, because of fully qualified symbol name change of frozen abi:

```patch
$ diff -U3 /tmp/abi8/*5rd8RyVSLH3hm12xJDVCJWgc1gyqb4Ukt2hJLJNfsB5v* /tmp/abi8/*ULV2jDndxR3JB677ayyjaamtAcZ24q75tCkHS2bKVoy*
--- /tmp/abi8/serde_snapshot__tests__test_bank_serialize__BankAbiTestWrapperFuture_frozen_abi__test_abi_digest_5rd8RyVSLH3hm12xJDVCJWgc1gyqb4Ukt2hJLJNfsB5v     2020-10-20 21:17:01.836997400 +0900
+++ /tmp/abi8/serde_snapshot__tests__test_bank_serialize__BankAbiTestWrapperFuture_frozen_abi__test_abi_digest_ULV2jDndxR3JB677ayyjaamtAcZ24q75tCkHS2bKVoy      2020-10-20 21:21:11.131409159 +0900
@@ -1617,11 +1617,11 @@
                         struct MessageProcessor (fields = 0)
             element solana_runtime::serde_snapshot::SerializableAccountsDB<solana_runtime::serde_snapshot::future::Context>
                 tuple (elements = 4)
-                    element solana_runtime::serde_snapshot::utils::serialize_iter_as_map::SerializableMappedIterator<core::iter::adapters::Map<core::slice::Iter<alloc::vec::Vec<alloc::sync::Arc<solana_runtime::accounts_db::AccountStorageEntry>>>, <solana_runtime::serde_snapshot::future::Context as solana_runtime::serde_snapshot::TypeContext>::serialize_accounts_db_fields<solana_sdk::abi_digester::AbiDigester>::{{closure}}>>
+                    element solana_runtime::serde_snapshot::utils::serialize_iter_as_map::SerializableMappedIterator<core::iter::adapters::Map<core::slice::Iter<alloc::vec::Vec<alloc::sync::Arc<solana_runtime::accounts_db::AccountStorageEntry>>>, <solana_runtime::serde_snapshot::future::Context as solana_runtime::serde_snapshot::TypeContext>::serialize_accounts_db_fields<solana_frozen_abi::abi_digester::AbiDigester>::{{closure}}>>
                         map (entries = 1)
                             key u64
                                 primitive u64
-                            value solana_runtime::serde_snapshot::utils::serialize_iter_as_seq::SerializableSequencedIterator<core::iter::adapters::Map<core::slice::Iter<alloc::sync::Arc<solana_runtime::accounts_db::AccountStorageEntry>>, <solana_runtime::serde_snapshot::future::Context as solana_runtime::serde_snapshot::TypeContext>::serialize_accounts_db_fields<solana_sdk::abi_digester::AbiDigester>::{{closure}}::{{closure}}>>
+                            value solana_runtime::serde_snapshot::utils::serialize_iter_as_seq::SerializableSequencedIterator<core::iter::adapters::Map<core::slice::Iter<alloc::sync::Arc<solana_runtime::accounts_db::AccountStorageEntry>>, <solana_runtime::serde_snapshot::future::Context as solana_runtime::serde_snapshot::TypeContext>::serialize_accounts_db_fields<solana_frozen_abi::abi_digester::AbiDigester>::{{closure}}::{{closure}}>>
                                 seq (elements = 1)
                                     element solana_runtime::serde_snapshot::future::SerializableAccountStorageEntry
                                         struct SerializableAccountStorageEntry (fields = 2)

```

Reply:
Perfect, thanks for confirming!